nixos/modules/services/networking/ddclient.nix

```diff
@@ -102,6 +102,22 @@ in
           Method to determine the IP address to send to the dynamic DNS provider.
         '';
       };
+
+      verbose = mkOption {
+        default = true;
```

Comment:
I have 15 lines in my log every 20 minutes. And since I am not looking into the logs every day I like to see thinks that break and not thinks that continuously report that everything is fine.
I can understand that during first setup the default behavior should be verbose. Therefore the current declaration sounds sane to me. But aside of that, I just wanted to add new options with this patch and don't change declaration behavior, so thinks stay the same for everybody else.